ros/src/foxglove_bridge/package.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<package format="3">
+    <name>foxglove_bridge</name>
+    <version>0.8.5</version>
+    <description>ROS Foxglove Bridge</description>
+    <license>MIT</license>
+    <url type="website">https://github.com/foxglove/ros-foxglove-bridge</url>
+    <author email="ros-tooling+foxglove_bridge@foxglove.dev">Foxglove</author>
+    <maintainer email="ros-tooling+foxglove_bridge@foxglove.dev">Foxglove</maintainer>
+
+    <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
+    <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake</buildtool_depend>
+
+    <!-- ROS 1 runtime dependencies -->
```

Comment:
some ROS 1 stuff here that could be cleaned up, or it can be done in a follow up PR